bin/fin

```diff
@@ -2855,6 +2856,68 @@ config_generate ()
 	fi
 }
 
+create_site ()
+{
+	while true; do
+		read -p "Please enter the name of your project (all one word, no spaces or slashes): " projectName
+		#todo, validate this input
+		break
+	done
+	echo ''
+
+	echo -e "What would you like to install?"
+	echo -e "${green}[1]${NC} Drupal 7"
+	echo -e "${green}[2]${NC} Drupal 8"
+	echo -e "${green}[3]${NC} Wordpress"
+	echo -e ""
+	while true; do
+		read -p ": " newSiteVersion
+		case ${newSiteVersion} in
+			1 )
+				newHumanReadableVersion="Drupal 7"
+				newSiteRepoName="drupal7"
+				break
+				;;
+			2 )
+				newHumanReadableVersion="Drupal 8"
+				newSiteRepoName="drupal8"
```

Comment:
I will make the change regarding storing URLs as variables, but I believe the change regarding `FIN_BRANCH` would break this command, because FIN_BRANCH has the value "develop".  None of the sample projects have a branch called "develop"